apps/etterlatte-saksbehandling-ui/client/src/components/behandling/BehandlingRoutes.tsx

```diff
@@ -139,7 +151,7 @@ export function revurderingRoutes(behandling: IBehandlingReducer): Array<Behandl
   const beregning = { path: 'beregne', description: 'Beregning', kreverBehandlingsstatus: IBehandlingStatus.BEREGNET }
   const defaultRoutes: Array<BehandlingRouteTypes> = erOpphoer(behandling.revurderingsaarsak!!)
     ? [revurderingsoversikt, vilkaarsvurdering, beregning]
-    : [revurderingsoversikt, vilkaarsvurdering, beregningsgrunnlag, beregning]
+    : [revurderingsoversikt, vilkaarsvurdering, trygdetid, beregningsgrunnlag, beregning]
```

Comment:
Hvis jeg forstår det riktig blir det slik at både trygdetid og vilkårsvurdering er mulige å trykke på tabsoversikten over, hvis man går videre fra saksoversikten, er det en sperre på å gå direkte fra trygdetid til beregningsgrunnlag uten å ha fylt ut vilkårsvurdering?

Reply:
Altså jeg ser at det er en isFailure på trygdetidssiden men kanskje skille på en ikke opprettet vilkårsvurdering og andre feil? Så får vi gitt saksbehandler en tydligere beskjed.

Reply:
Ikke helt, men du er inne på at det er noe som potensielt kan bli litt rart her i noen spesifikke scenarier hvor man hopper frem og tilbake.

Nå er det sånn at man fra søknadsoversikten kun kan gå til vilkårsvurdering (som tidligere), men etter å ha gjennomført en vilkårsvurdering kan man gå til både trygdetid og beregningsgrunnlag, siden de er "sidestilt". 

Det er kanskje fornuftig å kreve et separat behandlingssteg for trygdetid 🤔

Reply:
Ikke helt, men du er inne på at det er noe som potensielt kan bli litt rart her i noen spesifikke scenarier hvor man hopper frem og tilbake.

Nå er det sånn at man fra søknadsoversikten kun kan gå til vilkårsvurdering (som tidligere), men etter å ha gjennomført en vilkårsvurdering kan man gå til både trygdetid og beregningsgrunnlag, siden de er "sidestilt". 

Det er kanskje fornuftig å kreve et separat behandlingssteg for trygdetid 🤔

Reply:
Ser nå at de "rare" tilfelle er like feil i dagens løsning. 
Dersom man går til beregning, og tilbake til beregningsgrunnlag og endrer trygdetid, så kan man gå videre til brev/attestering uten å måtte beregne på nytt.. 

Tror kanskje det er en bug som bør løses i en egen PR?

Reply:
Det er vel to forskjellige problemstillinger
1. Er at trygdetid og beregningsgrunnlag begge blir klikkbare når vilkårsvurdering er ferdig
2. At man ikke oppdaterer behandlingsstatusen ved endringer på foregående sider, den er litt knotete da man fortsatt har data på noen sider selvom man ikke kan gå til de hvis man endrer. Det blir fort en veldig dårlig brukeropplevelse.

Reply:
1. Kan vel løses her med at den blir et eget steg mens 
2. Er jeg usikker på om vi vil gjøre noe med

Reply:
Høres bra ut

Reply:
Kommer endringer på denne.